dev/create-release/release-build.sh

```diff
@@ -181,7 +181,7 @@ BASE_PROFILES="-Pmesos -Pyarn -Pkubernetes"
 
 PUBLISH_SCALA_2_13=1
 SCALA_2_13_PROFILES="-Pscala-2.13"
-if [[ $SPARK_VERSION < "3.2" ]]; then
+if [ "$(echo $SPARK_VERSION | awk -F "." '{if ($1 <= 3 && $2 < 2) print $1}')" ]; then
```

Comment:
Or this too, was it nonstandard bash?

Reply:
SC2072 (error): Decimals are not supported. Either use integers only, or use bc or awk to compare.

Reply:
Hm, nothing here is actually using decimals - it's a lexicographic comparison. This alternative is less readable, but I also think it's wrong? "2.4" would not pass this check. We can revert and suppress this warning, or fix the logic to handle 3.x and <= 2.x possibilities separately in this clause

Reply:
There indeed has some proble here. `10.0` will not pass `$SPARK_VERSION < "3.2"`. The condition can be changed to `echo $SPARK_VERSION | awk -F "." '{if (($1 < 3) || ($1 == 3 && $2 < 2)) print $1}'`.
Hm, maybe skip the check and leave waning comment here?